tests/test_reporter.py

```diff
@@ -225,7 +225,7 @@ def test_submit_queue_full_batch_size_1(self):
 
     @gen_test
     def test_submit_batch_size_2(self):
-        reporter, sender = self._new_reporter(batch_size=2, flush=0.005)
+        reporter, sender = self._new_reporter(batch_size=2, flush=0.01)
```

Comment:
Why is this change required in this diff?

Reply:
the test was unstable when it relied on the difference between 1ms and 5ms